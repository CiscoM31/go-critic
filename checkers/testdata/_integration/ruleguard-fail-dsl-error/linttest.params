check -@ruleguard.rules rules1.go -@ruleguard.failOnError=dsl -enable ruleguard ./... | linttest.golden