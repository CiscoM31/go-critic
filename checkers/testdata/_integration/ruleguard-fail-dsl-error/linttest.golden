exit status 1
debug: ruleguard: initialization failure: ruleguard init error: typechecker error: rules1.go:16:4: m.AdvancedMatch undefined (type dsl.Matcher has no field or method AdvancedMatch)
init checkers: ruleguard init error: typechecker error: rules1.go:16:4: m.AdvancedMatch undefined (type dsl.Matcher has no field or method AdvancedMatch)